cinema/views.py

```diff
@@ -31,8 +41,27 @@ class CinemaHallViewSet(viewsets.ModelViewSet):
 
 
 class MovieViewSet(viewsets.ModelViewSet):
-    queryset = Movie.objects.all()
-    serializer_class = MovieSerializer
+    @staticmethod
+    def _params_to_ints(string_ids):
+        return [int(str_id) for str_id in string_ids.strip("/").split(",")]
```

Comment:
why do you need that `strip('/')` ?
And also this `staticmethod` is duplicated in two ViewSets. Maybe avoid this somehow?